test_helpers/index.go

```diff
@@ -73,25 +73,22 @@ func createEmptyApp(appName, cwd string) (string, error) {
 	return path.Join(parentDir, appName), nil
 }
 
-func killTestContainers(appDir, appName string) error {
-	composeProjectName := composebuilder.GetDockerComposeProjectName(appName)
-	writer := ioutil.Discard
-	err := application.CleanApplicationContainers(appDir, composeProjectName, writer)
+func killTestContainers(appDir string) error {
+	doesExist, err := util.DoesFileExist(path.Join(appDir, "application.yml"))
 	if err != nil {
-		return errors.Wrap(err, "Failed to kill test containers")
+		return err
 	}
-	testComposeProjectName := composebuilder.GetTestDockerComposeProjectName(appName)
-	err = application.CleanServiceTestContainers(appDir, testComposeProjectName, writer)
-	if err != nil {
-		return errors.Wrap(err, "Failed to kill test containers")
+	if doesExist {
+		cmdPlus := execplus.NewCmdPlus("exo", "clean") // nolint gas
```

Comment:
I don't think this is what we want to do here. `killTestContainers` is used for creating dangling images/volumes. Do those tests still pass with this new implementation?

Reply:
Auhh. I will revert the changes here and add a new function to clean up at the end of a scenario (which was the other place using this)